src/test/java/com/aquaq/training/test/MainTest.java

```diff
@@ -0,0 +1,176 @@
+
+
+        ByteArrayInputStream in = new ByteArrayInputStream(input.getBytes());
+        System.setIn(in);
+
+
+        Main.main(null);
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().contains(expectedString));
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().endsWith("Exiting..."));
+    }
+
+    @Test
+    public void testMainOption3(){
+        userString1 = "little";
+        String reverseWord = "elttil";
+        String expectedString = userString1 + " reversed is " + reverseWord;
+
+        String input = "3\n" + userString1 + "\n5\n";
+
+
+        ByteArrayInputStream in = new ByteArrayInputStream(input.getBytes());
+        System.setIn(in);
+
+
+        Main.main(null);
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().contains(expectedString));
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().endsWith("Exiting..."));
+    }
+
+    @Test
+    public void testMainOption4(){
+        userString1 = "Was it a car or a cat I saw?";
+        String expectedString = userString1 + " is a Palindrome";
+
+        String input = "4\n" + userString1 + "\n5\n";
+
+
+        ByteArrayInputStream in = new ByteArrayInputStream(input.getBytes());
+        System.setIn(in);
+
+
+        Main.main(null);
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().contains(expectedString));
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().endsWith("Exiting..."));
+    }
+
+    @Test
+    public void testMainOption5(){
+
+        String input = "5\n";
+
+
+        ByteArrayInputStream in = new ByteArrayInputStream(input.getBytes());
+        System.setIn(in);
+
+
+        Main.main(null);
+
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().endsWith("Exiting..."));
+    }
+
+    @Test
+    public void testMainDefaultNotNumber(){
+
+        String expectedString = "Invalid input, please enter a number from 1-5";
+
+        String input = "not a number\n5\n";
+
+
+        ByteArrayInputStream in = new ByteArrayInputStream(input.getBytes());
+        System.setIn(in);
+
+
+        Main.main(null);
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().contains(expectedString));
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().endsWith("Exiting..."));
+    }
+
+    @Test
+    public void testMainDefaultNumber(){
+
+        String expectedString = "Invalid input, please enter a number from 1-5";
+
+        String input = "7r\n5\n";
+
+
+        ByteArrayInputStream in = new ByteArrayInputStream(input.getBytes());
+        System.setIn(in);
+
+
+        Main.main(null);
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().contains(expectedString));
+
+        assertTrue(outputStreamCaptor.toString()
+                .trim().endsWith("Exiting..."));
+    }
```

Comment:
Think about adding an end to end test that goes through all available options then exits, mocking the return of StringHelper class's methods.